sympy/printing/aesaracode.py

```diff
@@ -69,6 +76,8 @@
             sympy.Transpose: DimShuffle((False, False), [1, 0]),
     }
 
+    aesara_version = tuple(getattr(aesara, '__version__').split('.'))
```

Comment:
I guess we don't need this any more.

Reply:
Good spot. Address in https://github.com/sympy/sympy/pull/24747/commits/72b658b0f8f70c78ba3d645a2dacfe3bbcbc6f90.